init.sh

```diff
@@ -87,28 +116,34 @@ cd "$DIR_LIB_GEN" && [ ! -f "$REPLACECALL_JAR" ] \
 #
 echo
 echo "Setting up Gradle dependencies ... "
-GRADLE_ZIP=defects4j-gradle.zip
-# The BSD version of stat does not support --version or -c
-if stat --version &> /dev/null; then
-    # GNU version
-    cmd="stat -c %Y $GRADLE_ZIP"
-else
-    # BSD version
-    cmd="stat -f %m $GRADLE_ZIP"
-fi
+
+GRADLE_DISTS_ZIP=defects4j-gradle-dists.zip
+GRADLE_DEPS_ZIP=defects4j-gradle-deps.zip
+
+dists_ts=$(get_modification_timestamp $GRADLE_DISTS_ZIP)
+deps_ts=$(get_modification_timestamp $GRADLE_DEPS_ZIP)
 
 cd "$DIR_LIB_GRADLE"
-if [ -e $GRADLE_ZIP ]; then
-    old_ts=$($cmd)
-else
-    old_ts=0
+
+old_dists_ts=0
+old_deps_ts=0
+
+if [ -e $GRADLE_DISTS_ZIP ]; then
+    old_dists_ts=$($dists_ts)
```

Comment:
The variable `dists_ts` holds the actual timestamp (not a cmd) and should not be evaluated twice -- right?